caffeine/inhibitors.py

```diff
@@ -71,7 +71,7 @@ def inhibit(self):
 
         self.__cookie = self.__proxy.Inhibit("Caffeine", dbus.UInt32(0),
                                              INHIBITION_REASON,
-                                             dbus.UInt32(12))
+                                             dbus.UInt32(4))
```

Comment:
Why this change?

Reply:
This was necessary, so that GnomeInhibitor doesn't keep the screen on. This way it prevents suspension, but allows the screen to be turned off.

Please check the documentation for the flags here: https://people.gnome.org/~mccann/gnome-session/docs/gnome-session.html#org.gnome.SessionManager.Inhibit